packages/create-block/lib/templates.js

```diff
@@ -161,9 +161,11 @@ const getPluginTemplate = async ( templateName ) => {
 
 	try {
 		if ( existsSync( resolve( templateName ) ) ) {
-			return await configToTemplate( require( resolve( templateName ) ) );
+			return await configToTemplate(
```

Comment:
Not leaving this as an official review, but `return await` doesn't do anything necessary. When returning a promise result you should just `return` the result. I see it was done in the original code, but it wouldn't hurt to remove it now while editing this code.